moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js

```diff
@@ -132,6 +134,21 @@ const DropdownMenu = ({
           Account
         </MenuItem>
         <Divider />
+        <MenuItem onClick={handleDropdownClose}>
+          <ListItemIcon>
+            <MapOutlinedIcon fontSize="small" />
+          </ListItemIcon>
+          <Link
+            href="https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=404d31d56b57491abe53ccfd718fcaee"
+            target="_blank"
+            rel="noopener noreferrer"
+            color="inherit"
+            underline="none"
+          >
+            ArcGIS Online map
```

Comment:
I wanted to doublecheck if this link should be added to the mobile menu too

Reply:
yeesh—thanks for flagging that! now that i'm staring at these two menus i think i'm going to merge the mobile into the dropdown menu. i'm not seeing a good reason to have two distinct menus anymore.

Reply:
cool! that sounds great to me. less chance for broken stuff in the future too! 🙏